doc/tutorial.md

```diff
@@ -1,33 +1,33 @@
 # Tutorial
-This tutorial intend to setup an entire dds and ros2 communication environment from zero.
-To archieve this, we need to deploy a `dds` world and a `ros2` world that will be connected thanks to `soss`.
+This tutorial intends to setup an entire dds and ros2 communication environment from zero.
+To achieve this, we need to deploy a `dds` world and a `ros2` world that will be connected thanks to `soss`.
 
-## Prerrequisites
+## Prerequisites
 ### Ros2
-Any version of `crystal`, `dashing`, or `eloquent` is valid for the propose of this tutorial.
+Any version of `ros2` (either `crystal`, `dashing`, or `eloquent`) is valid for the purpose of this tutorial.
 
 To install it, you can take a look to the
 [official documentation](https://index.ros.org/doc/ros2/Installation/Eloquent/Linux-Install-Binary/).
 
-In order to test it(supposing eloquent), please open two terminals and do the following:
+In order to test it (supposing eloquent), please open two terminals and do the following:
 - First terminal:
   ```
-  source ~/ros2_eloquent/ros2-linux/setup.bash
+  source ~/ros2_eloquent/install/setup.bash
```

Comment:
This seems to be so, but the previously linked page shows that path (https://index.ros.org/doc/ros2/Installation/Eloquent/Linux-Install-Binary/#sourcing-the-setup-script).

Maybe the tutorial is bad, did you check it?

Reply:
Yes I checked it. It doesn't work for me. On my pc it only works with "install" in spite of "ros2-linux", that's why I changed it.

Reply:
As we talk, this is due to the installation way, in this case, by binaries.